src/components/ambassadorCard/ambassadorCard.module.scss

```
@@ -132,15 +120,51 @@
     &::-webkit-scrollbar-track {
       background: $accent-color;
     }
-  }
-}
 
-.birthday::after {
-  content: url("../../assets/partyHat.png");
-  width: 4rem;
-  height: auto;
-  position: absolute;
-  left: 50%;
-  top: 0;
-  transform: translate(-50%, -90%);
+    h3 {
+      font-size: 0.9rem;
+    }
+
+    .mod {
+      display: flex;
+      gap: 0.5rem;
+      align-items: center;
+
+      img {
+        width: 1.5rem;
+        height: 1.5rem;
+        object-fit: cover;
+        border-radius: 0;
+      }
+    }
+
+    .compact {
+      display: flex;
+      flex-wrap: wrap;
+      gap: 0.25rem 1.5rem;
+    }
+
+    .info {
+      display: inline-flex;
+      align-items: center;
+      gap: 0.5rem;
```

```suggestion
      gap: 0.5rem;
      color: lighten($primary-color, 20%);
```

maybe do the lighten once and put it in a var